sunpy/io/file_tools.py

```diff
@@ -198,18 +163,19 @@ def detect_filetype(filepath):
     filetype : `str`
         The type of file.
     """
+    if "http" in filepath or "ftp" in filepath:
```

Comment:
This seems like a bad idea, what if I have a file at `/home/me/http/mymap.fits`? It would return `None` here because it has "http" in the filepath!

Reply:
Yea very true is guess `startswith('http://')` and `startswith('http://')` would be better but still not fool proof.